composer.json

```diff
@@ -7,6 +7,7 @@
     "prefer-stable": true,
     "require": {
         "php": ">=7.3",
+        "drupal/components": "^2.4",
```

Comment:
We actually don't need this module. Since we are placing the bcl files under the /template/bcl folder, you can access them by doing `@oe_bootstrap_theme/bcl`.

Reply:
Yep, my bad! I didn't see that, and forgot about oe_theme 😄  I'll create a custom twig loader for this project to expose this namespace. Enabling the whole component module to have just its loader is too much.

Reply:
Actually, the components module handles only namespaces, so it's the correct fit for this. 2x my bad 😄 
I'll restore the module.

Reply:
Removed the commit that was dropping the components module.